rllib/connectors/__init__.py

```diff
@@ -0,0 +1,35 @@
+from typing import Tuple, Any
+from ray.tune.registry import RLLIB_CONNECTOR, _global_registry
+
+from ray.util.annotations import PublicAPI
+from ray.rllib.connectors.connector import Connector, ConnectorContext
+
+
+@PublicAPI(stability="alpha")
+def register_connector(name: str, cls: Connector):
```

Comment:
define these 2 in registry.py and just import here?

Reply:
Tried that, it game me circular dependency issues :( I solved circular dependency by this. Basically the problem is that pipleineconnectors use register_connector and get_connector. I cannot import them in the `registry.py` if registry.py also includes `register_connector`. The `register_connector` API has to be imported before both of these connector apis.